code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -76,7 +77,7 @@
 		"Skrell" = 'icons/mob/species/skrell/suit.dmi',
 		"Vox" = 'icons/mob/species/vox/suit.dmi',
 		"Vulpkanin" = 'icons/mob/species/vulpkanin/suit.dmi',
-		"Drask" = 'icons/mob/species/drask/suit.dmi'
+		"Drask" = 'icons/mob/species/drask/suit.dmi',

```

Comment:
Are you using Github for desktop? If so just edit it again, commit it, then sync it.